go-controller/pkg/network-attach-def-controller/network_attach_def_controller.go

```diff
@@ -117,9 +125,11 @@ func (nadController *NetAttachDefinitionController) Start() error {
 		return err
 	}
 
-	err = nadController.networkManager.Start()
-	if err != nil {
-		return err
+	if nadController.networkManager != nil {
```

Comment:
In which cases do we have a nil `networkManager`?  Is this test specific?

Reply:
this was originally added so that I could start a nadcontroller without a network manager for CNI. Now that the nadcontroller from the node is extended to CNI, I can probably remove it.